packages/cozy-client/src/models/contact.js

```diff
@@ -134,22 +135,21 @@ export const getDisplayName = contact =>
  * @returns {string} - the contact's 'byFamilyNameGivenNameEmailCozyUrl' index
  */
 export const getIndexByFamilyNameGivenNameEmailCozyUrl = contact => {
-  const indexByFamilyNameGivenNameEmailCozyUrl = capitalize(
-    [
-      get(contact, 'name.familyName', ''),
-      get(contact, 'name.givenName', ''),
-      getPrimaryEmail(contact),
-      getPrimaryCozyDomain(contact)
-    ]
-      .join('')
-      .trim()
-  )
+  const indexByFamilyNameGivenNameEmailCozyUrl = [
+    get(contact, 'name.familyName', ''),
+    get(contact, 'name.givenName', ''),
+    getPrimaryEmail(contact),
+    getPrimaryCozyDomain(contact)
+  ]
+    .join('')
+    .trim()
+    .toLowerCase()
```

Comment:
I haven't understood well why it was necessary to lowercase here ? I thought that makeCaseInsensitiveStringSorter would already do this. Is it to fix a bug ? What was the behavior ?

Reply:
This is to be consistent between the data in couchDB and the data in cozy-client, in order to have as much as possible an identical sorting behavior.

Reply:
Is it to fix a bug ? What was the behavior ?

Reply:
Description updated to be more explicit